src/main/java/exception/LineSplitter.java

```diff
@@ -0,0 +1,5 @@
+package exception;
```

Comment:
why do you put this class into exception package? Please pay attention to your package structure